common/source/docs/common-geofencing-landing-page.rst

```diff
@@ -32,5 +34,7 @@ Inclusion and Exclusion Fences are defined easily in Mission Planner under the P
     Inclusion/Exclusion Fence Failsafe <common-polygon_fence>
 [/site]
 [site wiki="plane"]
-    Geo-Fencing Failsafes <geofencing>
+    Cylindrical Fence Failsafe (version 4.1 and later) <common-ac2_simple_geofence>
+    Inclusion/Exclusion Fence Failsafe (version 4.1 and later)<common-polygon_fence>
```

Comment:
Not familiar with the syntax here, is a space required after the close bracket and open arrow?
here --> `... later)<common-...`

```suggestion
    Inclusion/Exclusion Fence Failsafe (version 4.1 and later) <common-polygon_fence>
```

Reply:
no...spaces ignored